python/lsst/meas/base/references.py

```diff
@@ -51,25 +53,14 @@ class BaseReferencesTask(lsst.pipe.base.Task):
 
     Parameters
     ----------
-    schema : `lsst.afw.table.Schema`, optional
+    schema : `lsst.afw.table.Schema`
         The schema of the reference catalog.
-    butler : `lsst.daf.persistence.butler.Butler`, optional
-        A butler that will allow the task to load its schema from disk.
+    butler : `None`.
+        Compatibility parameter. Should always be `None`.
 
     Notes
     -----
-    At least one of the ``schema`` and ``butler`` arguments must be present;
-    if both are, ``schema`` takes precedence.
-
-    ``BaseReferencesTask`` defines the required API for the references task,
-    which consists of:
-
-    - ``getSchema(butler)``
-    - ``fetchInPatches(butler, tract, filter, patchList)``
-    - ``fetchInBox(self, butler, tract, filter, bbox, wcs)``
-    - the ``removePatchOverlaps`` config option
-
-    It also provides the ``subset`` method, which may be of use to derived
+    Provides the ``subset`` method, which may be of use to derived
```

Comment:
The line below mentions `fetchInBox`, but you removed that.  Can the `subset` method go too?

Reply:
I was waiting for someone to tell me that the entire `references.py` file can be removed because it no longer seems to do anything and was seemingly entirely gen2 in practice.